packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -43,10 +43,7 @@ module.exports = React.createClass({
     editor: React.PropTypes.bool,
     fixture: React.PropTypes.string,
     fullScreen: React.PropTypes.bool,
-    proxies: React.PropTypes.arrayOf(React.PropTypes.shape({
-      component: React.PropTypes.func.isRequired,
-      ref: React.PropTypes.string,
-    })),
+    proxies: React.PropTypes.arrayOf(React.PropTypes.func),
```

Comment:
Do all three types of React components match `React.PropTypes.func`? (React.createClass, ES6Class and stateless functions)